site/content/cluster/kubernetes_cluster.md

```diff
@@ -0,0 +1,95 @@
+{{% /notice %}}
+
+
+## Prerequisites
+
+-   A running Kubernetes cluster.
+    -   For local testing, you can use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), [Docker desktop](https://www.docker.com/products/docker-desktop), or [we have a script](https://raw.githubusercontent.com/m3db/m3db-operator/master/scripts/kind-create-cluster.sh) you can use to start a 3 node cluster with [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+-   To use the M3DB operator chart, you need [Helm](https://helm.sh/)
+
+## Create An etcd Cluster
+
+M3 stores its cluster placements and runtime metadata in [etcd](https://etcd.io) and needs a running cluster to communicate with.
+
+We have example services and stateful sets you can use, but feel free to use your own configuration and change any later instructions accordingly.
+
+```shell
+kubectl apply -f https://github.com/m3db/m3db-operator/blob/v0.9.0/example/etcd/etcd-minikube.yaml
+```
+
+If the etcd cluster is running on your local machine, update your _/etc/hosts_ file to match the domains specified in the `etcd` `--initial-cluster` argument. For example to match the `StatefulSet` declaration in the _etcd-minikube.yaml_ above, that is:
+
+```text
+$(minikube ip) etcd-0.etcd
+$(minikube ip) etcd-1.etcd
+$(minikube ip) etcd-2.etcd
+```
+
+Verify that the cluster is running with something like the Kubernetes dashboard, or the command below:
+
+```shell
+kubectl exec etcd-0 -- env ETCDCTL_API=3 etcdctl endpoint health
+```
+
+## Install the Operator with Helm
+
+Add the M3 operator repository:
+
+```shell
+helm repo add m3db https://m3-helm-charts.storage.googleapis.com/stable
+```
+
+Install the M3 operator chart:
+
+```shell
+helm install m3db-operator m3db/m3db-operator
+```
+
+## Create an M3 Cluster
+
+The following creates an M3 cluster with 3 replicas of data across 256 shards that connects to the 3 available etcd endpoints.
+
+It creates three isolated groups for nodes, each with one node instance. In a production environment you can use a variety of different options to define how nodes are spread across groups based on factors such as resource capacity, or location.
+
+It creates namespaces in the cluster with the `namespaces` parameter. You can use M3-provided presets, or define your own. This example creates a namespace with the `10s:2d` preset.
+
+The cluster derives pod identity from the `podIdentityConfig` parameter, which in this case is the UID of the Pod.
+
+[Read more details on all the parameters in the Operator API docs](https://operator.m3db.io/api/).
+
+```shell
+kubectl apply -f https://github.com/m3db/m3db-operator/blob/v0.9.0/example/m3db-local.yaml
```

Comment:
Also 404 not found, I believe we probably want this instead:
https://github.com/m3db/m3db-operator/blob/master/example/m3db-cluster.yaml

Reply:
Yes, this is one I created myself, it's mentioned in the comments, and I am waiting for https://github.com/m3db/m3db-operator/pull/244